graylog2-server/src/main/java/org/graylog2/inputs/InputServiceImpl.java

```diff
@@ -135,6 +135,16 @@ public List<Input> allByType(final String type) {
         return inputs.build();
     }
 
+    @Override
+    public List<String> findIdsByTitle(String title) {
+        final ImmutableList.Builder<String> inputIds = ImmutableList.builder();
+        for (final DBObject o : query(InputImpl.class, new BasicDBObject(MessageInput.FIELD_TITLE, title))) {
```

Comment:
`from_input` uses the `InputRegistry` and also case insensitive compares, so the result might differ.